src/fits.cc

```diff
@@ -52,7 +52,7 @@ std::string makeLimitedFitsHeaderImpl(std::vector<std::string> const &paramNames
         auto name = (lastPeriod == std::string::npos) ? fullName : fullName.substr(lastPeriod + 1);
         std::type_info const &type = metadata.typeOf(name);
 
-        std::string out = "";
+        std::string out;
```

Comment:
I'd suggest keeping these in. While the default values for standard C++ types are fairly intuitive, I don't see any harm in being explicit.

Reply:
Sounds good.